public/app/features/explore/state/query.ts

```diff
@@ -1227,3 +1270,60 @@ export const processQueryResponse = (
     clearedAtIndex: state.isLive ? state.clearedAtIndex : null,
   };
 };
+
+export function reloadCorrelations(exploreId: string): ThunkResult<void> {
```

Comment:
Not for this PR but I wonder if we should create `correlations.ts` with all correlations related actions, this file is getting very big.

Reply:
I did it anyway 👼